src/software/proto/message_translation/defending_side.cpp

```diff
@@ -0,0 +1,9 @@
+#include "software/proto/message_translation/defending_side.h"
+
+std::unique_ptr<DefendingSideProto> createDefendingSideProto(bool defending_positive_side)
```

Comment:
über nit: we don't add `Proto` to the end of any of the other functions in `message_translation`

Reply:
fixed